cli/azd/pkg/infra/provisioning/bicep/prompt.go

```diff
@@ -103,7 +103,23 @@ func (a *BicepProvider) locationsWithQuotaFor(
 	ctx context.Context, subId string, locations []string, quotaFor []string) ([]string, error) {
 	var sharedResults sync.Map
 	var wg sync.WaitGroup
+
+	azureAiServicesLocations, err := a.azureClient.GetResourceSkuLocations(
+		ctx, subId, "AIServices", "S0", "Standard", "accounts")
+	if err != nil {
+		return nil, fmt.Errorf("getting Azure AI Services locations: %w", err)
+	}
+
+	if locations == nil {
+		// If no locations are provided, use the Azure AI Services locations
+		locations = azureAiServicesLocations
+	}
+
 	for _, location := range locations {
+		if !slices.Contains(azureAiServicesLocations, location) {
+			// Skip locations that are not in the list of Azure AI Services locations
```

Comment:
Is `azureAiServicesLocations` expected to be a reliable source of truth here, when `locations` is provided? Just wondering how the "S0" "Standard" sku factors in here